designsafe/apps/projects/models/agave/experimental.py

```diff
@@ -62,19 +62,44 @@ class Experiment(RelatedEntity):
     project = fields.RelatedObjectField(ExperimentalProject)
     dois = fields.ListField('Dois')
 
-    def to_datacite_json(self):
+    def to_datacite_json(self, project={}):
```

Comment:
Passing lists/dicts as default arguments can lead to strange behavior: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments 

This PR doesn't mutate the project that gets passed in, but someone might come into the codebase later and accidentally do something weird.

Reply:
I updated the code: 
```
def to_datacite_json(self, project=None):
   if project is None:
       project={}
```